admin/cordova_sandbox.sb

```diff
@@ -0,0 +1,106 @@
+(version 1) ; Fri Jan  6 12:01:53 2017
+
+; I M P O R T A N T:
+; /private/tmp/foobar below needs to be changed with the folder that holds the source code for Cordova project
```

Comment:
I think is better to use those sandbox files (xcodebuild, cocoapods, cordova, etc) are going to be imported/included in other sandbox files that sets the access to the working directory.